c/src/ml-api-service.c

```diff
@@ -355,13 +372,19 @@ _ml_service_offloading_create_json (ml_service_s * mls, JsonObject * object)
     _ml_error_report_return (status, "Failed to create ml-option.");
   }
 
-  status = _ml_service_offloading_conf_to_opt (object, "offloading", option);
+  status = ml_service_offloading_create (mls);
+  if (status != ML_ERROR_NONE) {
+    _ml_error_report ("Failed to create ml-service-offloading.");
+  }
+
+  status =
+      _ml_service_offloading_conf_to_opt (mls, object, "offloading", option);
   if (status != ML_ERROR_NONE) {
     _ml_error_report ("Failed to set ml-option from config file.");
     goto done;
   }
 
-  status = ml_service_offloading_create (mls, option);
+  status = ml_service_offloading_start (mls, option);
   if (status != ML_ERROR_NONE) {
     _ml_error_report ("Failed to create ml-service-offloading.");
```

Comment:
Let's update the error message

Reply:
Sorry, code is changed in last pr (https://github.com/nnstreamer/api/pull/480/commits/421be49e2f24f0089e665a6850f97f0f3ebe6bbc)